lib/Dialect/Torch/Transforms/ReduceOpVariants.cpp

```diff
@@ -117,6 +117,30 @@ class ConvertToImmutableTensors : public RewritePattern {
 };
 } // namespace
 
+// Reduce Ops without value semantics but the corresponding without trailing
+// underscore variant doesn't exist.
+namespace {
+class ReduceNonValueSemanticOps : public RewritePattern {
+public:
+  ReduceNonValueSemanticOps(MLIRContext *context)
+      : RewritePattern(MatchAnyOpTypeTag(), /*benefit=*/1, context) {}
+  LogicalResult matchAndRewrite(Operation *op,
+                                PatternRewriter &rewriter) const override {
+    Operation *newOp;
```

Comment:
nit: This line is not needed. You can do `Operation *newOp = rewriter....`, that way all the code related to `newOp` is together